README.md

```diff
@@ -71,7 +71,8 @@ your config/application.rb to apply RuboCop autocorrection to code generated by
 module YourCoolApp
   class Application < Rails::Application
     config.generators.after_generate do |files|
-      system("bundle exec rubocop -A --fail-level=E #{files.shelljoin}", exception: true)
+      filtered_files = files.reject { |file| file.end_with?('.erb') }
+      system("bundle exec rubocop -A --fail-level=E #{filtered_files.shelljoin}", exception: true)
```

Comment:
There may be other than `.erb` depending on a template generator (e.g. `.haml`). How about limiting it to `.rb` only?

```suggestion
      ruby_files = files.filter { |file| file.end_with?('.rb') }
      system("bundle exec rubocop -A --fail-level=E #{ruby_files.shelljoin}", exception: true)
```

Reply:
I think boths options have their drawback. I believe one will always need to modify it to fit ones needs.

If I would go for `filter` instead of `reject`, I would need to filter for the following files from `.rubocop.yml`:

```yaml
AllCops:
  Include:
    - '**/*.builder'
    - '**/*.rake'
    - '**/*.rb'
    - '**/*.ru'
    - '**/config.ru'
    - '**/Gemfile'
    - '**/Rakefile'
```

Reply:
Right. However `.rb` will suffice for most of the code created by `bin/rails g`. So, the intention will be sufficiently conveyed I think :-)

Reply:
@tmaier friendly ping :-)

Reply:
Done :)